platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/model/constants/AppConstant.java

```diff
@@ -0,0 +1,6 @@
+package com.mapbox.mapboxsdk.testapp.model.constants;
+
+public class AppConstant {
+
+    public final static int STYLE_VERSION = 9;
```

Comment:
I will add a unit test but ideally I'm taking up https://github.com/mapbox/mapbox-gl-native/issues/1462 in the near future.